tooling-tests/src/test/resources/test-src/generated-sources/tool_test_client_7/helloWorldInputMessageOutputEmpty_pb.bal

```diff
@@ -37,7 +37,7 @@ public client class TestInputStructNoOutputStreamingClient {
         if response is () {
             return response;
         } else {
-            [anydata, map<string|string[]>] [payload, headers] = response;
+            _ = response;
         }
     }
```

Comment:
@dilanSachi better if we can remove this entire else block since it is unnecessary. This won’t be a high priority task. Let's create an issue to keep track of that. We can improve that as a separate task.

Reply:
https://github.com/ballerina-platform/ballerina-standard-library/issues/2442